python/grass/pygrass/vector/testsuite/test_vector3d.py

```diff
@@ -19,11 +19,12 @@
 
 def generate_coordinates(number, bbox=None, with_z=False):
     """Return 2 or 3 random arrays of coordinates"""
+    rng = np.random.default_rng()
```

Comment:
In tests, shouldn't we be using a fixed seed? (Not this PR that causes this)

Reply:
You have a point. I can add a seed. Not sure what consequences that may have though.

Reply:
Well, if the tests were passing before without a seed, they'll continue passing after. 

But it's weird that having different values each time didn't affect the tests themselves. That means it doesn't check correctly or precisely what it needs at some point.

Reply:
That is because the set of completely random points are first written to a map, and then read that same map again to then checked that there was no change during writing and reading,.. So it seems OK, no?